### PR TITLE
Add the Redirect module to Lightning

### PR DESCRIPTION
**Pull request by kdruckman**
add redirect module

### CODE REVIEW

**Comment by phenaproxima** [comment]
I think you might need to get in sync with the upstream remote, since you're still working from a fork. Do these instructions help? https://help.github.com/en/articles/syncing-a-fork

If not, let's handle this after scrum. :)

File: tests/src/Kernel/Update405Test.php [hunk 1]
```diff
@@ -21,16 +21,34 @@ class Update405Test extends KernelTestBase {
...
+    $moduleHandler = $this->container->get('module_handler');
+
+    $this->assertTrue($moduleHandler->moduleExists('autosave_form'));
+    $this->assertTrue($moduleHandler->moduleExists('conflict'));
+  }
+
+  /**
+   * @covers ::enableRedirect
+   */
+  public function testEnableRedirect() {
```
Comment (phenaproxima) [inline tests/src/Kernel/Update405Test.php]: So...this actually doesn't need to be in its own method. I would suggest we rename testEnableAutosaveForm() to testUpdate(), and have it do something like this:

```php
$moduleHandler = $this->container->get(...);

$this->assertFalse($moduleHandler->moduleExists('autosave_form'));
// Same thing for Conflict and Redirect

$update = Update405::create($this->container);
$update->enableAutosaveForm();
$update->enableRedirect();

$moduleHandler = ...
$this->assertTrue($moduleHandler->moduleExists('autosave_form'));
// ...etc.
```
This allows us to test everything we need to, and keep things a bit simpler and less verbose. :)

**Review by phenaproxima — CHANGES_REQUESTED** [review]
This will need a CHANGELOG entry, and it's blocked on a new release of Lightning Core (which will require an other go of `composer update`). Otherwise, it looks excellent overall; I would like a small optimization in the test. :)

File: tests/src/Kernel/Update405Test.php (hunk 1 above)
Comment (kdruckman) [reply tests/src/Kernel/Update405Test.php]: makes sense!

File: tests/src/Kernel/Update405Test.php [hunk 2]
```diff
@@ -19,18 +19,25 @@ class Update405Test extends KernelTestBase {
...
   /**
    * @covers ::enableAutosaveForm
+   * @covers ::enableRedirect
    */
-  public function testEnableAutosaveForm() {
-    /** @var \Drupal\Core\Extension\ModuleHandlerInterface $module_handler */
-    $module_handler = $this->container->get('module_handler');
-    $this->assertFalse($module_handler->moduleExists('autosave_form'));
-    $this->assertFalse($module_handler->moduleExists('conflict'));
+  public function testUpdate() {
```
Comment (phenaproxima) [inline tests/src/Kernel/Update405Test.php:24]: I think that there might be an extra blank line here that shouldn't be.

**Review by phenaproxima — APPROVED** [review]
I literally only found one _very_ minor thing wrong with this. It looks perfect otherwise.

File: tests/src/Kernel/Update405Test.php (hunk 2 above)
Comment (kdruckman) [reply tests/src/Kernel/Update405Test.php:24]: I'll get those coding standards things figured out asap. :)

**Comment by phenaproxima** [comment]
This worked great under quick manual testing. The CI failures are due to Chrome being a pain in the ass, and are not related. I ran the test locally and confirmed it passes. This is done, and I'm merging it. First-class work, @kdruckman, thanks!

**Pull request merged**